zoomx/src/main/java/com/zoomx/zoomx/view/request/RequestViewHolder.java

```diff
@@ -0,0 +1,54 @@
+package com.zoomx.zoomx.view.request;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+
+import com.zoomx.zoomx.R;
+import com.zoomx.zoomx.model.RequestEntity;
+import com.zoomx.zoomx.util.ColorUtils;
+import com.zoomx.zoomx.util.FormatUtil;
+
+/**
+ * Created by Ibrahim AbdelGawad on 12/13/2017.
+ */
+
+public class RequestViewHolder extends RecyclerView.ViewHolder {
+    private Context context;
+    TextView urlTextView, codeTextView, methodTypeTextView, startDateTextView;
+
+
+    public RequestViewHolder(Context context , View view) {
+        super(view);
+
+        this.context = context;
+        urlTextView = view.findViewById(R.id.url_tx);
+        codeTextView = view.findViewById(R.id.code_txt);
+        methodTypeTextView = view.findViewById(R.id.method_txt);
+        startDateTextView = view.findViewById(R.id.date_txt);
+    }
+
+    public void bind(final RequestEntity requestEntity , final OnRequestItemClickListener onRequestItemClickListener)
+    {
```

Comment:
Can you please reformat the code before push?

Reply:
Sure, Sorry!